leaderboard.py

```diff
@@ -49,71 +47,38 @@ def compute_elo(battles, k=4, scale=400, base=10, initial_rating=1000):
     rating[model_a] += k * (scored_point_a - expected_score_a)
     rating[model_b] += k * (1 - scored_point_a - expected_score_b)
 
-  return rating
-
-
-def get_docs(tab: str,
-             summary_lang: str = None,
-             source_lang: str = None,
-             target_lang: str = None):
-  if tab == LeaderboardTab.SUMMARIZATION:
-    collection = db.collection("arena-summarizations").order_by("timestamp")
-
-    if summary_lang:
-      collection = collection.where(filter=base_query.FieldFilter(
-          "model_a_response_language", "==", summary_lang.lower())).where(
-              filter=base_query.FieldFilter("model_b_response_language", "==",
-                                            summary_lang.lower()))
-
-    return collection.stream()
+  return {model: math.floor(rating + 0.5) for model, rating in rating.items()}
```

Comment:
is `math.floor(rating + 0.5)` tried to do rounding? how about `round(rating)`

Reply:
the `round` function rounds numbers to the nearest even number so I used `math.floor(rating + 0.5)` instead so numbers will be rounded to the nearest even/odd number

Reply:
I don't see anything about choosing the nearest even number
https://docs.python.org/3/library/functions.html#round

Reply:
round(2.5) is five
> Note The behavior of [round()](https://docs.python.org/3/library/functions.html#round) for floats can be surprising: for example, round(2.675, 2) gives 2.67 instead of the expected 2.68. This is not a bug: it’s a result of the fact that most decimal fractions can’t be represented exactly as a float. See [Floating-Point Arithmetic: Issues and Limitations](https://docs.python.org/3/tutorial/floatingpoint.html#tut-fp-issues) for more information.